### PR TITLE
Bug 1753156: Fix the trusted-ca bundle filename

### PR DESCRIPTION
**Pull request by stlaz**
cc @mfojtik

### CODE REVIEW

File: bindata/v4.1.0/kube-apiserver/pod.yaml [hunk 1]
```diff
@@ -34,9 +34,9 @@ spec:
     command: ["/bin/bash", "-ec"]
     args:
         - |
-          if [ -f /etc/kubernetes/static-pod-resources/kube-apiserver-certs/configmaps/trusted-ca-bundle/tls-ca-bundle.pem ]; then
+          if [ -f /etc/kubernetes/static-pod-resources/kube-apiserver-certs/configmaps/trusted-ca-bundle/ca-bundle.crt ]; then
```
Comment (xingxingxia) [inline bindata/v4.1.0/kube-apiserver/pod.yaml]: In the proxy CA env I tested https://bugzilla.redhat.com/show_bug.cgi?id=1753156#c0 , it shows:
oc rsh -n openshift-kube-apiserver kube-apiserver-ip-10-0-139-96.sa-east-1.compute.internal
sh-4.2# ls /etc/kubernetes/static-pod-resources/kube-apiserver-certs/configmaps/trusted-ca-bundle/ca-bundle.crt
ls: cannot access /etc/kubernetes/static-pod-resources/kube-apiserver-certs/configmaps/trusted-ca-bundle/ca-bundle.crt: No such file or directory
sh-4.2# ls /etc/kubernetes/static-pod-resources/kube-apiserver-certs
ls: cannot access /etc/kubernetes/static-pod-resources/kube-apiserver-certs: No such file or directory
sh-4.2# ls -l /etc/kubernetes/static-pod-resources/
total 8
drwxr-xr-x. 10 root root  224 Sep 18 08:47 configmaps
-rw-r--r--.  1 root root 6185 Sep 18 08:47 kube-apiserver-pod.yaml
drwxr-xr-x.  5 root root   97 Sep 18 08:47 secrets

Comment (stlaz) [reply bindata/v4.1.0/kube-apiserver/pod.yaml]: oh yeah, I did not notice, the whole path is wrong, not just the name, you saved the day I believe

**Pull request merged**